pennylane/utils.py

```diff
@@ -393,3 +393,54 @@ def expand_vector(vector, original_wires, expanded_wires):
     )
 
     return qml.math.reshape(expanded_tensor, 2**M)
+
+
+def prod_sort(op_list, wire_map: dict = None):
```

Comment:
Not sure if this is the place for this function.

Reply:
I remember @josh146 said he was looking into removing the `utils.py` file.

Reply:
100% support the removal of this file.  If this is going to be used in operator arithmetic right now primarily, I vote we move it there. Even just making it a local, private helper until we have a compelling reason to generalize it.

Reply:
Moved function to `prod.py`

Reply:
Agree, `utils.py` is where functionality goes to be forgotten 😆